tools/clang_tidy/lib/clang_tidy.dart

```diff
@@ -344,11 +344,23 @@ class ClangTidy {
 
   Future<int> _runJobs(List<WorkerJob> jobs) async {
     int result = 0;
-    final Set<String> pendingJobs = <String>{for (final WorkerJob job in jobs) job.name};
-
-    void reporter(int totalJobs, int completed, int inProgress, int pending, int failed) {
-      return _logWithTimestamp(ProcessPool.defaultReportToString(
-        totalJobs, completed, inProgress, pending, failed));
+    final Set<String> pendingJobs = <String>{
+      for (final WorkerJob job in jobs) job.name
+    };
+
+    void reporter(
```

Comment:
I accidentally formatted the whole doc.  Here is the actual fix though.  I just brought over the implementation since they removed `defaultReportToString`.